data/card_catalogue/card-manipulation.json

```diff
@@ -4,77 +4,101 @@
+        "passive": ""
+      },
       "flavour": "You never know what he'll pull out of thin air.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Brainstorm.png"
     }
   },
   {
     "title": "Spark of Inspiration",
     "upgrade": "Randomize",
     "tier": "red",
     "details": {
-      "rules":"Cast two random Red Cards.",
-      "flavour": "I've got an idea",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "rules": {
+        "active":"Draw 2 red cards. Cast both.",
+        "passive": ""
+      },
+      "flavour": "I've got an idea...",
+      "image": "assets/core/card/illustration/Spark of Inspiration.png"
     }
   },
   {
     "title": "Randomize",
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules": "Cast two random Gold Cards.",
+      "rules": {
+        "active": "Draw 2 gold cards. Cast both.",
+        "passive": ""
+      },
       "flavour": "I don't know what I want, but I do want a lot of it.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Randomize.png"
     }
   },
   {
     "title": "Grobble",
     "upgrade": "Slipshod Sorcery",
     "tier": "green",
     "details": {
-      "rules": "Draw 6 random Grey Cards.",
+      "rules": {
+        "active": "Draw 6 grey cards.",
+        "passive": ""
+      },
       "flavour": "Oh man, I've really Grobbled this one.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Grobble.png"
     }
   },
   {
     "title": "Slipshod Sorcery",
     "upgrade": "Spectrum Dance",
     "tier": "red",
     "details": {
-      "rules": "Place 10 blue cards onto the table. Cast up to 8 of those Cards.",
+      "rules": {
+        "active": "Draw 10 blue cards, face-up. Cast up to 8 and discard the rest.",
+        "passive": ""
+      },
       "flavour": "What we lack in quality we've made up with quantity.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Slipshod Sorcery.png"
     }
   },
   {
     "title": "Spectrum Dance",
     "upgrade": "",
     "tier": "gold",
     "details": {
-      "rules":"Place 5 cards of each rarity onto the table. Cast one Card of each rarity.",
+      "rules": {
+        "active":"Draw 5 cards of each rarity, face-up. Cast one of each rarity and discard the rest.",
+        "passive": ""
+      },
       "flavour": "The greatest plans begin with simple tactics.",
-      "image": "assets/core/card/illustration/Bean Blast.png"
+      "image": "assets/core/card/illustration/Spectrum Dance.png"
     }
   },
   {
     "title": "Imitate",
     "upgrade": "Copy",
     "tier": "blue",
     "details": {
-      "rules": "Copy the last Blue card used by any character",
+      "rules": {
+        "active": "Copy the effect of the last blue card Cast by any character.",
```

Comment:
changed to specify that it's the effect that' you're copying and not the card itself